MAUI/Autocomplete/UI-Customization.md

```diff
@@ -890,7 +890,7 @@ The following image illustrates the result of the above code:
 
 ### TextAlignment
```

Comment:
Why did you adding TextAlignment title inside the border visibility? Ensure in other editor controls.